package.json

```diff
@@ -4,8 +4,13 @@
   "description": "Some sample web app.",
   "main": "index.html",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "live-server"
+    "build-task:scss-compile": "node-sass-chokidar --source-map true _scss/style.scss -o _temp",
```

Comment:
`build-task:scss-compile` Runs `node-sass-chokidar` which will take our _scss/style.scss file and compile it into _temp/style.css.